doc/api/addons.markdown

```diff
@@ -345,9 +345,12 @@ using v8::Value;
 
 void RunCallback(const FunctionCallbackInfo<Value>& args) {
   Isolate* isolate = args.GetIsolate();
+  if (!args[0]->IsFunction()) {
+    return;
```

Comment:
I was thinking the same thing since it's done below, but in terms of a learning experience it may be better to introduce these things at a slower pace? I don't mind either way tbh.